dashboard/app/models/stage.rb

```diff
@@ -136,7 +136,7 @@ def lockable_state(students)
     end
     script_level = self.script_levels[0]
     return students.map do |student|
-      user_level = UserLevel.find_by(user_id: student.id, level: script_level.level, script_id: self.script.id)
+      user_level = student.last_attempt_for_any script_level.levels, script_id: self.script.id
```

Comment:
A little, but that change wasn't supposed to be in this PR. I'll tag you in the PR where it belongs, once I create it.